packages/build-config/shims/worker-shim.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+require('babel-polyfill');
+
+(function execute() {
+  var entryPoint = require('hops-worker-entry-point');
+  if (entryPoint.__esModule) {
```

Comment:
Should probably also be `typeof entryPoint !== 'function' && typeof entryPoint.default === 'function'`